apps/ligo-virgo/src/pages/workspace/docs/components/toc/TOC.tsx

```diff
@@ -112,15 +115,19 @@ const renderTOCContent = tocDataSource => {
     );
 };
 
-export const TOC = (props: TOCProps) => {
-    const { editor } = props;
+export const TOC = () => {
     const { page_id } = useParams();
     const [tocDataSource, setTocDataSource] = useState<TOCType[]>([]);
     const [activeBlockId, setActiveBlockId] = useState('');
 
     /* store page/block unmount-listener */
     const listenerMapRef = useRef<ListenerMap>(new Map());
 
+    const { currentEditors } = useCurrentEditors();
+    const editor = useMemo(() => {
+        return currentEditors[page_id] as Virgo;
+    }, [currentEditors, page_id]);
+
```

Comment:
This `memo` is unnecessary

```suggestion
    const editor = currentEditors[page_id] as Virgo;
```